tf-deployer/vsphere_named_machines.yaml

```diff
@@ -53,19 +53,60 @@ items:
       datacenter_id = "${data.vsphere_datacenter.dc.id}"
     }
 
+    data "template_file" "cloud_provider_config" {
+      template = <<EOF
+    [Global]
+    datacenters = "$${datacenter}"
+    insecure-flag = "1" #set to 1 if the vCenter uses a self-signed cert
```

Comment:
Do we expect any usage of non-self-signed certs? If so, both here and `allow_unverified_ssl = true` above would need to be changed together, right? Probably not worth changing until we need to, but seems like a sharp edge we are leaving for the first person that wants to bring a real cert.